ui/lesson/ScriptingChallenge/Runner/Terminal.tsx

```diff
@@ -57,11 +76,13 @@ function Terminal({ className }: { className?: string }, ref) {
                         break
                       }
                       case 'error': {
-                        output.innerHTML += "<div class='error'>Error: "+payload+"</div>"
+                        document.body.classList.add("error-background")
+                        output.innerHTML += "<div class='error'>"+payload+"</div>"
                         output.parentElement.scrollTop = output.scrollHeight
                         break
                       }
                       case 'success': {
+                        document.body.classList.add("success-background")  
```

Comment:
need to remove error before this to render the green background
```suggestion
                        document.body.classList.remove("error-background")
                        document.body.classList.add("success-background")
```